version.json

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
You need to mention this file in [Dockerfile](https://github.com/keepers-team/webtlo/blob/41553607ef873a22396aa9fa1a27faaa32d09cdb/Dockerfile#L47) too.

Reply:
@Evgen-BioNik Please, replace `version.txt` with `version.json` in `Dockerfile`.